### ISSUE TITLE
Accepting different kinds of zettel formats

### ISSUE THREAD

**Issue by tevino** [Zettelkasten-Method/zkviz#24]
Currently zkviz requires the zettel to be formatted like this:

- Filename: `YYYYMMDDHHMMSS TITLE.md`
- The `YYYYMMDDHHMMSS` is unique

or it finds nothing, say [this case](https://github.com/Zettelkasten-Method/zkviz/issues/22) for example.

It would be great if there's an option to choose another format of zettel:

`> zkviz --zettel-format [TS-TITLE|ID]`

Users that are not using a timestamp-based ID could benefit from zkviz too.

Here's my setup: http://tevinzhang.com/digital-zettelkasten/

TL;DR:

- Filename: `ID.md`
- Title: in the file starting with `# `

The current implementation is called as`TS-TITLE` since it is a combination of a timestamp and title, the format I described above is `ID`, the names may vary, here's just a proposal to add a new flag indicating different formats.

I could submit a PR if that's needed.
